### PR TITLE
docs(autok3s): update CN and EN readme files

### PR DESCRIPTION
**Pull request by yzeng25**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,12 +28,43 @@ AutoK3s is a lightweight tool for simplifying the cluster management of K3s, it
...
+| `-d`                         | To enable debug mode.                                                                |
+| `-p`                         | To specify which cloud provider to use, please see the chart below for more details. |
+| `--name`                     | To specify the name of the cluster.                                                  |
+| `--master`                   | To specify the number of master nodes that you want to create or add.                |
+| `--worker`                   | To specify the number of master nodes that you want to create or add.                |
+| `--registry`                 | To specify the private registry for `containerd`.                                    |
+| `--cloud-controller-manager` | To enable Kubernetes Cloud-Controller-Manager component.                             |
+| `--ui`                       | To enable Kubernetes Dashboard UI component.                                         |
+| `--terway 'eni'`             | To enable CNI network components(for Alibaba only).                                  |
```
Comment (Jason-ZW) [inline README.md:59]: Remove these params

File: README.md [hunk 2]
```diff
@@ -23,12 +28,43 @@ AutoK3s is a lightweight tool for simplifying the cluster management of K3s, it
...
+- `autok3s create`: To create and lauch a K3s cluster
+- `autok3s join`：To add a node or multiple nodes for an existing K3s cluster.
+
+## Frequently-used Parameters
+
+These parameters are frequently used in AutoK3s:
+
+| Parameters                   | Description                                                                          |
+| :--------------------------- | :----------------------------------------------------------------------------------- |
+| `-d`                         | To enable debug mode.                                                                |
```
Comment (Jason-ZW) [inline README.md:51]: --debug or -d

File: README.md [hunk 3]
```diff
@@ -23,12 +28,43 @@ AutoK3s is a lightweight tool for simplifying the cluster management of K3s, it
...
+| `-p`                         | To specify which cloud provider to use, please see the chart below for more details. |
```
Comment (Jason-ZW) [inline README.md:52]: --provider or -p

File: README.md [hunk 4]
```diff
@@ -47,7 +83,21 @@ Autok3s can run in two different modes: Local mode and Rancher mode.
...
 In this mode, you can use Autok3s via CLI or a local UI.
 
-The following commands use the `alibaba` provider, please check the [prerequisites](docs/i18n/en_us/alibaba/README.md) before you run that:
+#### CLI
+
+The following commands are examples for creating a K3s cluster and adding nodes for an existing cluster in `alibaba` provider. Please make sure all [prerequisites](docs/i18n/en_us/alibaba/README.md) are met before executing these commands.
+
+##### Creating a K3s Cluster
+
+You can use `autok3s create` command to create a K3s cluster. A typical `autok3s create` command usually includes the following elements:
```
Comment (Jason-ZW) [inline README.md:92]: following args?

File: README.md [hunk 5]
```diff
@@ -56,6 +106,24 @@ export ECS_ACCESS_KEY_SECRET='<Your secret access key>'
 autok3s -d create -p alibaba --name myk3s --master 1 --worker 1
 ```
 
+##### Adding Nodes for an Existing K3s Cluster
+
+You can use `autok3s join` command to add nodes for an existing K3s cluster. A typical `autok3s join` command usually includes the following elements:
```
Comment (Jason-ZW) [inline README.md:111]: following args?

File: docs/i18n/zh_cn/README.md [hunk 6]
```diff
@@ -23,31 +27,73 @@ AutoK3s是用于简化K3s集群管理的轻量级工具，您可以使用Autok3s
...
+| :--------------------------- | :---------------------------------------------- |
+| `-d`                         | 开启 debug 模式。                               |
+| `-p`                         | provider，即云服务提供商，详情参考下表。        |
+| `--name`                     | 指定将要创建的集群的名称。                      |
+| `--master`                   | 指定创建的 master 节点数量 。                   |
+| `--worker`                   | 指定创建的 worker 节点数量 。                   |
+| `--registry`                 | 配置`containerd`私有镜像仓库。                  |
+| `--cloud-controller-manager` | 开启 Kubernetes Cloud-Controller-Manager 组件。 |
+| `--ui`                       | 开启 Kubernetes Dashboard UI 组件。             |
+| `--terway 'eni'`             | 开启公有云 CNI 网络插件（仅适用于阿里云）。     |
```
Comment (Jason-ZW) [inline docs/i18n/zh_cn/README.md:56]: remove these flags

File: docs/i18n/zh_cn/README.md [hunk 7]
```diff
@@ -23,31 +27,73 @@ AutoK3s是用于简化K3s集群管理的轻量级工具，您可以使用Autok3s
...
+- `autok3s create`：创建和启动 K3s 集群。
+- `autok3s join`：为已有的 K3s 集群添加节点。
+
+## 常用参数
+
+AutoK3s 命令中常用的参数如下表所示：
+
+| 参数                         | 描述                                            |
+| :--------------------------- | :---------------------------------------------- |
+| `-d`                         | 开启 debug 模式。                               |
```
Comment (Jason-ZW) [inline docs/i18n/zh_cn/README.md:48]: `--debug` or `-d`

File: docs/i18n/zh_cn/README.md [hunk 8]
```diff
@@ -23,31 +27,73 @@ AutoK3s是用于简化K3s集群管理的轻量级工具，您可以使用Autok3s
...
+| `-p`                         | provider，即云服务提供商，详情参考下表。        |
```
Comment (Jason-ZW) [inline docs/i18n/zh_cn/README.md:49]: `--provider` or `-p`

**Review by Jason-ZW — CHANGES_REQUESTED** [review]

File: README.md [hunk 9]
```diff
@@ -23,12 +28,43 @@ AutoK3s is a lightweight tool for simplifying the cluster management of K3s, it
...
+
+## Frequently-used Parameters
+
+These parameters are frequently used in AutoK3s:
+
+| Parameters                   | Description                                                                          |
+| :--------------------------- | :----------------------------------------------------------------------------------- |
+| `-d`                         | To enable debug mode.                                                                |
+| `-p`                         | To specify which cloud provider to use, please see the chart below for more details. |
+| `--name`                     | To specify the name of the cluster.                                                  |
```
Comment (JacieChao) [inline README.md:53]: --name or -n

File: docs/i18n/zh_cn/README.md [hunk 10]
```diff
@@ -23,31 +27,73 @@ AutoK3s是用于简化K3s集群管理的轻量级工具，您可以使用Autok3s
...
+| `--name`                     | 指定将要创建的集群的名称。                      |
```
Comment (JacieChao) [inline docs/i18n/zh_cn/README.md:50]: --name or -n

Comment (yzeng25) [reply docs/i18n/zh_cn/README.md:50]: fixed

File: README.md (hunk 9 above)
Comment (yzeng25) [reply README.md:53]: fixed

File: docs/i18n/zh_cn/README.md (hunk 8 above)
Comment (yzeng25) [reply docs/i18n/zh_cn/README.md:49]: fixed

File: docs/i18n/zh_cn/README.md (hunk 7 above)
Comment (yzeng25) [reply docs/i18n/zh_cn/README.md:48]: fixed

File: docs/i18n/zh_cn/README.md (hunk 6 above)
Comment (yzeng25) [reply docs/i18n/zh_cn/README.md:56]: fixed

File: README.md (hunk 5 above)
Comment (yzeng25) [reply README.md:111]: fixed

File: README.md (hunk 4 above)
Comment (yzeng25) [reply README.md:92]: fixed

File: README.md (hunk 3 above)
Comment (yzeng25) [reply README.md:52]: fixed

File: README.md (hunk 2 above)
Comment (yzeng25) [reply README.md:51]: fixed

File: README.md (hunk 1 above)
Comment (yzeng25) [reply README.md:59]: fixed

**Review by Jason-ZW — APPROVED** [review]

**Pull request merged**